docs/user/reference/destinations.md

```diff
@@ -0,0 +1,19 @@
+# Destinations
+
```

Comment:
Say a sentence or two about what a destination is? (Something along the lines of: "Destinations combine a preposition and a target, to define a place and behavior for a Cursorless action.")